app/views/journal_entries/show.html.erb

```diff
@@ -18,9 +18,12 @@
     <h2><%= t('journal_entry.content') %>:</h2>
     <p><%= @journal_entry.content %></p>
 
-    <%= link_to t('journal_entry.edit_entry'), edit_journal_entry_path(@journal_entry), class: 'button' %>
+    <%= link_to t('journal_entry.edit_entry'), edit_user_journal_entry_path(current_user, @journal_entry), class: 'button' %>
     <br>
-    <%= link_to t('journal_entry.back_to_entries'), journal_entries_path, class: 'button' %>
+    <%= link_to t('journal_entry.download'), user_journal_entry_path(current_user, @journal_entry, format: :pdf), class: 'button', target: '_blank' %>
```

Comment:
Maybe let's add a button for this

Reply:
ok thank you for your sugesstion i did

Reply:
<img width="274" alt="image" src="https://github.com/tgi-wtag/Journey-tg/assets/141820846/e7afb2df-5987-4b18-a6db-f7abf2b205b0">